waiter/src/waiter/util/utils.clj

```diff
@@ -332,6 +342,7 @@
   "Converts the error-context to the response body html string."
   [error-context render-fn]
   (-> error-context
+    (update :message escape-html)
```

Comment:
This needs to exclude changing the `&` in links.

Reply:
We should still HTML encode the `&` in the links, it still works even if something like `<a href"http://google.com/&amp;>"`. (source [https://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href](https://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href)). I'll add a unit test for this case